inputfiles/addedTypes.json

```diff
@@ -462,6 +462,13 @@
         ]
     },
     {
+        "kind": "method",
```

Comment:
this should come before the string overload. overload resolution is order dependent and `string` is a supertype of all literal strings.